spec/fixtures/Puppetfile

```diff
@@ -1,4 +1,5 @@
 mod 'homebrew',    '0.0.16', :github_tarball => 'boxen/puppet-homebrew'
+mod 'brewcask',    '0.0.6',  :github_tarball => 'toolbear/puppet-brewcask'
```

Comment:
why not use the original upstream repository - `boxen/puppet-brewcask` instead of your fork?